From 52e231b04635400292179cf51b30d7d9b6323fb2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mantas=20Mikul=C4=97nas?= <grawity@gmail.com>
Date: Mon, 8 Sep 2014 22:53:39 +0300
Subject: [PATCH] rules: net-setup-link - remove stray linebreak

If not backslash-escaped, it splits the rule in two.
---
 rules/80-net-setup-link.rules | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/rules/80-net-setup-link.rules b/rules/80-net-setup-link.rules
index 27c43b9437..420769497f 100644
--- a/rules/80-net-setup-link.rules
+++ b/rules/80-net-setup-link.rules
@@ -4,8 +4,7 @@ SUBSYSTEM!="net", GOTO="net_setup_link_end"
 
 IMPORT{builtin}="path_id"
 
-ACTION=="move", IMPORT{db}="ID_NET_DRIVER", IMPORT{db}="ID_NET_LINK_FILE",
-IMPORT{db}="ID_NET_NAME"
+ACTION=="move", IMPORT{db}="ID_NET_DRIVER", IMPORT{db}="ID_NET_LINK_FILE", IMPORT{db}="ID_NET_NAME"
 
 ACTION!="add", GOTO="net_setup_link_end"
 
